gspread/worksheet.py

```diff
@@ -1351,10 +1357,22 @@ def insert_row(self, values, index=1, value_input_option=ValueInputOption.raw):
             See `ValueInputOption`_ in the Sheets API.
 
         .. _ValueInputOption: https://developers.google.com/sheets/api/reference/rest/v4/ValueInputOption
+        :param bool inherit_from_before: (optional) If true, the new row will
+            inherit its properties from the previous row. Defaults to False,
+            meaning that the new row acquires the properties of the row
+            immediately after it.
         """
```

Comment:
this is a nice description, we should mention that we can't use this option for the row 1 (first row).